doc/source/user_reference/output_directory.rst

```diff
@@ -0,0 +1,132 @@
+.. _output_directory_structure:
+
+Output Directory Structure
+==========================
+
+This is an over overview of WA output directory structure.
+
+.. note:: In addition to files and subdirectories described here,
+          other content may present int the output directory for
+	  a run, depending on the enabled augmentations.
+
+Overview
+--------
+
+The output directory will contain a subdirectory for every job iteration was
+run, as well as some additional entries.  The following diagram illustrates the
+typical structure of WA output directory::
+
+	wa_output/
+	├── __meta/
+	│   ├── config.json
+	│   ├── jobs.json
+	│   ├── raw_config
+	│   │   ├── cfg0-config.yaml
+	│   │   └── agenda.yaml
+	│   ├── run_info.json
+	│   └── target_info.json
+	├── __failed/
+	│   └── wk1-dhrystone-1-attempt1
+	├── wk1-dhrystone-1/
+	│   └── result.json
+	├── wk1-dhrystone-2/
+	│   └── result.json
+	├── wk2-memcpy-1/
+	│   └── result.json
+	├── wk2-memcpy-2/
+	│   └── result.json
+	├── result.json
+	└── run.log
+
+This is the directory structure that would be generated after running two
+iterations each of ``dhrystone`` and ``memcpy`` workloads with no augmentations
+enabled, and with the first attempt of the first iteration of dhrystone having
```

Comment:
at the first iteration

Reply:
I believe "of" is correct here; "at" precedes a verb:

https://www.englishforums.com/English/AttemptToVsAttemptAt/wzrwh/post.htm

Reply:
Erm, never mind, I got that exactly the wrong way round.